announcement/announcement-api/api/src/java/org/sakaiproject/announcement/api/AnnouncementService.java

```diff
@@ -123,7 +123,10 @@ public interface AnnouncementService extends MessageService
     public static final String EVENT_ANNC_UPDATE_AVAILABILITY = SECURE_ANNC_ROOT + "revise.availability";
 
     public static final String MOTD_TOOL_ID = "sakai.motd";
-    
+
+	/** SAK-44622 **/
+	public static final String EVENT_AVAILABLE_ANNC = SECURE_ANNC_ROOT + "available.announcement";
```

Comment:
Indentation is off here.

Reply:
I am not exactly sure what you mean

Reply:
the formatting seems a bit weird, not lined up with line 125

Reply:
done, realigned line